server/db/models/conversation.js

```diff
@@ -22,4 +31,44 @@ Conversation.findConversation = async function (user1Id, user2Id) {
   return conversation;
 };
 
+Conversation.markAsRead = async function (conversationId, userId) {
+  const conversation = await Conversation.findOne({
+    where: {
+      id: {
+        [Sequelize.Op.eq]: conversationId
+      },
+      // Result should be null if userId provided is not in channel:
+      [Sequelize.Op.or]: [
+        {
+          user1Id:
+          {
+            [Sequelize.Op.eq]: userId
+          }
+        },
+        {
+          user2Id:
+          {
+            [Sequelize.Op.eq]: userId
+          }
+        }
+      ]
+    }
+  });
+
+  if (!conversation) throw 'conversationId not valid';
```

Comment:
You should just throw an error here 🙂 

```suggestion
  if (!conversation) throw new Error('conversationId not valid');
```